deploy/playbooks/roles/celery/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: pip install celery
+  pip:
+    name: "celery[librabbitmq]"
```

Comment:
since chacra is a *service* I think it is time for us to get a requirements.txt file and use that for our dependencies.